android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Mar 17 10:42:36 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
```

Comment:
I'll check this too.

Perhaps what I'll do will be to merge some of your changes.

Reply:
tried this with no luck, all sorts of issues with android studio.